requirements/app/test.txt

```diff
@@ -9,3 +9,4 @@ httpx
 trio
 pympler
 psutil
+setuptools==59.5.0
```

Comment:
Pinning setuptools here seems a little extreme. Minor differences on different platforms or environments would leave the dependency resolver with no room to resolve.

Reply:
I agree, I'm going to experiment with a wider range.

Reply:
I think the easiest is to pin the max version:

```suggestion
setuptools<=59.5.0
```
Then dependabot can send PRs to update it, and we can accept or reject depending if the version is good or not.